app/views/bulk_submissions/_upload.html.erb

```diff
@@ -3,7 +3,6 @@
   <strong>none</strong> of them will be processed. You will get feedback about any problems.</p>
 <% end %>
 <%= semantic_form_for @bulk_submission, html: { multipart: true } do |form|  %>
-  <%= form.semantic_errors :base %>
```

Comment:
Note: This removes the duplicate listing of the same error messages at the form level because they are already listed at the input control level.